hugo.yaml

```diff
@@ -140,7 +140,7 @@ params:
   github_branch: main
 
   # Google Custom Search Engine ID. Remove or comment out to disable search.
-  gcs_engine_id: 674c0c3807d874d09
+  # gcs_engine_id: 674c0c3807d874d09
```

Comment:
I suspect we should remove this so we don't expose the ID since we are not currently using it.

Reply:
```suggestion
```